bootstrap.js

```diff
@@ -230,8 +230,6 @@ function startup(aData, aReason) {
 function shutdown(aData, aReason) {
   // No need to do extra work here
   Log.debug("shutdown, aReason=", aReason);
-  // Load imports for the BOOTSTRAP_REASONS constants; should have happened before in most cases
-  loadImports();
```

Comment:
I'll look into it tonight! :)

On Tue, Apr 28, 2015 at 2:12 PM, pianoslum <notifications@github.com> wrote:

> In bootstrap.js
> <https://github.com/protz/GMail-Conversation-View/pull/952#discussion_r29289217>
> :
>
> > @@ -230,8 +230,6 @@ function startup(aData, aReason) {
> >  function shutdown(aData, aReason) {
> >    // No need to do extra work here
> >    Log.debug("shutdown, aReason=", aReason);
> > -  // Load imports for the BOOTSTRAP_REASONS constants; should have happened before in most cases
> > -  loadImports();
>
> I don't think a new pdf version is too important...
> What about the keyconfig-bug? If you knew the right Listener it could be
> fixed quite quickly...
> Otherwise: MERGE! ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/protz/GMail-Conversation-View/pull/952/files#r29289217>
> .
>